first homework/task2.js

```diff
@@ -1,21 +1,20 @@
 import csv from 'csvtojson';
-import { writeFile } from 'fs';
+import { createWriteStream } from 'fs';
 const csvFilePath='csv\\nodejs-hw1-ex1.csv';
 
-var data='';
+let stream=createWriteStream('test.txt');
 csv()
 .fromFile(csvFilePath)
-.on('error',(err)=>{
-	console.log(err)
-})
-.then((jsonObj)=>{
-	jsonObj.forEach((line)=>{
-                data+=JSON.stringify(line)+'\n';
-        })
+.subscribe((json,lineNumber)=>{
+                let outputLine={};
```

Comment:
Use features of es5+ to rewrite generation of the _outputLine_:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries
https://www.w3schools.com/js/js_es5.asp#mark_array_reduce
https://javascript.info/destructuring-assignment


To read:
https://www.w3schools.com/js/js_es5.asp
https://www.w3schools.com/js/js_es6.asp